src/ssh/Makefile-ssh.am

```diff
@@ -81,8 +98,15 @@ EXTRA_DIST += \
 	src/ssh/mock-bin/sss_ssh_knownhostsproxy \
 	src/ssh/mock-config \
 	src/ssh/invalid_known_hosts \
+	src/ssh/mock_known_hosts_2 \
 	$(NULL)
 
+if !HAVE_SSH_SESSION_HAS_KNOWN_HOSTS_ENTRY
+EXTRA_DIST += \
+	src/ssh/mock_known_hosts_2 \
+	$(NULL)
+endif
```

Comment:
This is duplicated and should be removed. EXTRA_DIST should never be conditionalized on configure tests, this defines what goes into our release tarballs.